### DISCUSSION TITLE
Can not install in win7 service pack1

### DISCUSSION THREAD

**Discussion by chancezheng** [wixtoolset/issues#7283] [Questions]
**Encountered at:** wixtoolset/repo@sha
when I install the package in win7, errored:
Failed to create the managed bootstrapper application
failed to create UX
failed to load UX
failed while running
**Detail:**
UI:WPF, （.net48）
bootstrapper.config : supportedRuntime:v4.0

I wang to know what's up.

and when I changed wpf targetframework to net6.0-windows, there is same error as above

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh]. You might also consider asking your question on the [wix-users mailing list][ml] or posting on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[ml]: http://wixtoolset.org/documentation/mailinglist/
[so]: https://stackoverflow.com/questions/tagged/wix
